format/FormatMultiImage.py

```diff
@@ -228,29 +228,30 @@ def get_imageset(
                 indices=single_file_indices,
             )
 
+            if single_file_indices is None:
+                single_file_indices = list(range(format_instance.get_num_images()))
```

Comment:
I think that now you are using `enumerate` below, this doesn't need to be converted to a list?
```suggestion
                single_file_indices = range(format_instance.get_num_images())
```

Reply:
it's iterated over potentially more than once, so no.

Reply:
In Python 3 `range` is apparently not an "iterator", so can be iterated over multiple times:

https://treyhunner.com/2018/02/python-range-is-not-an-iterator/

In Python 2, `range` returns a list, so this would also be fine.

Reply:
Nice. I stand corrected

Reply:
Done